start_dev

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+docker-compose -f docker-compose.yml -f docker-compose.dev.yml down
+cp dev.env .env 
+docker-compose -f docker-compose.yml -f docker-compose.dev.yml up --build
+docker-compose exec api sh -c "rails db:migrate RAILS_ENV=development"
```

Comment:
When I ran this script, the last line was never called because the process on line 5 never finished.

Reply:
This is a mistake on my end, I have moved migrations to an entry point file that is run every time the container starts so we no longer have to explicitly run them with a docker command.